frontend/appflowy_flutter/lib/util/expand_views.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:appflowy/workspace/application/view/view_bloc.dart';
+
+/// the key is view id
+final Map<String, Set<ViewBloc>> _recordBlocs = {};
+
+bool isViewExpanded(String id) => getViewBloc(id)?.state.isExpanded ?? false;
+
+void recordBloc(String id, ViewBloc bloc) {
+  final set = _recordBlocs[id] ?? {};
+  set.add(bloc);
+  _recordBlocs[id] = set;
+}
+
+void eraseBloc(String id, ViewBloc bloc) {
+  final set = _recordBlocs[id] ?? {};
```

Comment:
I think `set` is a poor name, maybe just `record` or similar.

Reply:
You are right, I'll rename them